tests/Util/ArrTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Ahc\Phint\Test;
+
+use Ahc\Phint\Util\Arr;
+use PHPUnit\Framework\TestCase;
+
+class ArrTest extends TestCase
+{
+    public function arrProvider()
+    {
+        return [
+            [[6, 7, 8], [2, 3, [4, 5]], [2, 3, [4, 5]]],
+            [[2, 3, [4, 5]], [2, 3, [4, 5]], [2, 3, [4, 5]]],
```

Comment:
should we have one more test case in this array: i.e. with stringy keys and deep nested values?

Reply:
Can you show some test cases about the nested array you mention?